### ISSUE TITLE
Using the all the phone screen adds a margin

### ISSUE THREAD

**Issue by k2helix** [crackededed/Xtra#469]
I noticed that when playing in full screen mode, there's a button to scale the stream to use all the phone screen.

However, unlike other apps such as YouTube (in which you can do this by zooming in the player), the video stream does not occupy the whole screen but only until the camera position, leaving a black margin between the phone's border and the position of the camera.

Here's an example of what I'm saying:![Screenshot_20240406-163042.png](https://github.com/crackededed/Xtra/assets/49949593/6236ac1c-5600-4da2-b6bc-3a8df79923e2)

**Comment by crackededed** [comment]
Settings -> Ignore display cutouts -> Always

**Comment by k2helix** [comment]
You're right! Sorry, I did not see that option, I thought if it was implemented it would be in the Player section, but it's literally the second option. Thanks for your help!
